model/permission.go

```diff
@@ -300,6 +331,24 @@ func initializePermissions() {
 		"authentication.permissions.manage_others_emojis.description",
 		PERMISSION_SCOPE_TEAM,
 	}
+	PERMISSION_CREATE_EMOJIS = &Permission{
+		"create_emojis",
+		"authentication.permissions.create_emojis.name",
+		"authentication.permissions.create_emojis.description",
+		PERMISSION_SCOPE_TEAM,
+	}
+	PERMISSION_DELETE_EMOJIS = &Permission{
+		"delete_emojis",
+		"authentication.permissions.delete_emojis.name",
+		"authentication.permissions.delete_emojis.description",
+		PERMISSION_SCOPE_TEAM,
+	}
+	PERMISSION_DELETE_OTHERS_EMOJIS = &Permission{
+		"delete_others_emojis",
+		"authentication.permissions.delete_others_emojis.name",
+		"authentication.permissions.delete_others_emojis.description",
+		PERMISSION_SCOPE_TEAM,
+	}
```

Comment:
I notice you don't remove the old legacy permissions from this file. I can think of arguments both ways, but curious what your reasoning was?

Reply:
Yes, I was trying to be cautious here about removing stuff that can break the app. but I think make sense to remove it, if something breaks with this, the tests and the compiler should complains about it. I going to try to remove it and see what happens.

Reply:
Fixed

Reply:
:+1: